pkg/cmd/pulumi/plugin/plugin_rm.go

```diff
@@ -60,7 +60,7 @@ func newPluginRmCmd() *cobra.Command {
 			// Parse the filters.
 			var kind apitype.PluginKind
 			var name string
-			var version *semver.Range
+			var version *semver.Version
```

Comment:
The description of the command says: "VERSION may be a range". Is that not going to work anymore?

Reply:
Oh I missed that... I'll change this one back then, it'll still have issues with metadata ranges but can probably just tell people not to do that.